core/templates/services/playthrough.service.ts

```diff
@@ -274,19 +283,23 @@ export class PlaythroughService {
    * If none of the issue types have been discovered, returns null instead.
    */
   private createNewPlaythrough(): Playthrough | null {
-    if (this.misTracker && this.misTracker.foundAnIssue()) {
+    if (this.misTracker.foundAnIssue()) {
```

Comment:
Hi @JeeveshGarg -- I was chatting with @vojtechjelinek about this (I was wondering whether it's actually safe to drop the check) and he suggests that there is still a possibility of things being null and that the usage of `!` might be wrong.

@vojtechjelinek knows more about this than I do (atm), so I'll defer to him to comment further. (@vojtechjelinek, just a thought -- if this ends up resulting in a general rule, we should probably add it to the wiki.)

Reply:
Done